README.markdown

```diff
@@ -51,23 +50,30 @@ registry_value { 'HKLM\System\CurrentControlSet\Services\Puppet\Description':
 }
 ```
 
+### Manage a single Registry value with a complex name
```

Comment:
We jump right into the "complex name" example after the most basic example, though it's unclear what "complex" means here without a little more explanatory copy.

Should we start with a simpler example to bridge the gap?  For instance, instead of going straight to something with a `path` to the value as an attribute, could we start with no `path` attribute and an explanation of what it does?

```puppet
registry_value { 'HKLM\System\CurrentControlSet\Services\Puppet\\\ValueWithA\Backslash':
  ensure     => present,
  type       => string,
  data       => "The Puppet Agent service periodically manages your configuration",
}
```

Or something without the extra slashes that shows off `path`?

```puppet
registry_value { 'MyValue' :
  path         => 'HKLM\System\CurrentControlSet\Services\Puppet\Value'
  ensure     => present,
  type       => string,
  data       => "The Puppet Agent service periodically manages your configuration",
}
```

The main thing I'm getting it is that it's unclear from the examples when / why you need to separate things out.

Reply:
What about

---

### Manage a single Registry value

``` puppet
registry_value { 'HKLM\System\CurrentControlSet\Services\Puppet\Description':
  ensure => present,
  type   => string,
  data   => "The Puppet Agent service periodically manages your configuration",
}
```

### Manage a single Registry value with a backslash in the value name

``` puppet
registry_value { 'HKLM\System\CurrentControlSet\Services\Puppet\\\ValueWithA\Backslash':
  ensure     => present,
  type       => string,
  data       => "The Puppet Agent service periodically manages your configuration",
}
```

### Manage a single Registry value with a different resource title

``` puppet
registry_value { 'PuppetDescription':
  path       => 'HKLM\System\CurrentControlSet\Services\Puppet\Description',
  ensure     => present,
  type       => string,
  data       => "The Puppet Agent service periodically manages your configuration",
}
```
```

Reply:
Yeah, example  seems reasonable... think we need a little explanation between each? Thoughts @michaeltlombardi / @RandomNoun7 as "new users"?

Reply:
From a pseudo-beginner perspective, these all look fine. I'm able to pick up the difference between title and path from context even if I don't know about how Puppet works under-the-hood.

The other two are reasonably obvious examples, even for me. 😄

Reply:
pushed

Reply:
These examples seem good to me. Very clear.